### DISCUSSION TITLE
How to exit a code block once you entered?

### DISCUSSION THREAD

**Discussion by yajo** [outline/outline#3087] [General] [answered]
1. Create a tip.
2. Insert a code block inside.
3. How to exit the block and continue writing in the tip?

https://user-images.githubusercontent.com/973709/153396642-66c30d99-8f63-40db-9a12-110169d098fe.mp4

I tried clicking in between the block and tip, hitting enter multiple times, alt+enter, ctrl+enter... Can't do it!

**Comment by tommoor** [comment, marked as answer]
Code blocks within tips isn't really a usecase we've optimized for – typically you exit a code block with the down arrow, however in this case it will lead you to exit the tip as well!

I think we could add support for ctrl/cmd+enter to exit the block without too much trouble.
